pkg/cli/aws/provider.go

```diff
@@ -16,22 +16,44 @@ limitations under the License.
 
 package aws
 
+// AwsCredentialKind - Aws credential kinds supported.
+type AwsCredentialKind string
```

Comment:
What is the convention in our repository for `Aws`? Should it be `AWS` or do we use both `Aws` and `AWS`? I believe we should use `AWS`.

Reply:
I see we are using mix of these 2 formats, mostly i see Aws in ucp code and AWS on the cli side. I have updated it to AWS format